DOL.WHD.Section14c.Web/webpack.config.js

```diff
@@ -57,7 +58,16 @@ module.exports = {
           'sass?sourceMap&includePaths[]=' + bourbon
         ]
       },
-      { test: /\.css$/, loader: 'style-loader!css-loader' },
+      {
+        test: /\.css$/,
+        exclude: helpers.root('src', 'v4'),
```

Comment:
This is excluded because the `angular2-template-loader` is adding these files to the bundle?

Reply:
@mgwalker these edits look good and won't interfere with anything I'm doing on the jenkins branch. Although webpack was bundling everything successfully, I was still having trouble loading the application in the browser. Adding the angular2-template-loader to the webpack config fixed this issue. I've checked this pull request out locally and everything looks good.